### PR TITLE
Implement CrisisModal when CrisisButton is clicked

### PR DESCRIPTION
**Pull request by ajaguasin**

### CODE REVIEW

File: js/components/CrisisModal/CrisisModal.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          </TouchableOpacity>
+        </LinearGradient>
+      </View>
+    </Modal>
+  );
+}
+
+CrisisModal.propTypes = {};
+
+export default CrisisModal;
```
Comment (siddharthparmar7) [inline js/components/CrisisModal/CrisisModal.js:51]: Add prop type validation, please.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes. Thank you.

**Comment by ajaguasin** [comment]
Changes resolved, thank you

File: js/screens/CommunityQuestion/CommunityQuestion.js [hunk 2]
```diff
@@ -18,19 +19,27 @@ const CommunityQuestion = props => {
...
             </TouchableOpacity>
           </LinearGradient>
 
+          <CrisisModal
+            toggleVisibility={props.toggleVisibility}
+            isVisible={props.isVisible}
+          />
           <CommunityQuestionForm navigation={props.navigation} />
         </View>
       </ScrollView>
```
Comment (siddharthparmar7) [inline js/screens/CommunityQuestion/CommunityQuestion.js:45]: you are receiving more props than just a navigation object. Please add prop type validation for all of your props.

**Review by siddharthparmar7 — APPROVED** [review]
Nice work. Please make the changes in your next pull request.

**Pull request merged**